portfolio/src/main/java/com/google/sps/data/Marker.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** Represents a marker on the map. */
+public class Marker {
```

Comment:
Not sure if you can access it from your project, but if you'd like, I believe you could use an existing map api's class for latitude/longitude tuples at https://github.com/googlemaps/google-maps-services-java/blob/b0a48afef86b2b6431821ea7d0f4c441674957f0/src/main/java/com/google/maps/model/LatLng.java#L24

It's compatible with some other API code in the same package which can let you do some cool geo stuff if you feel like it, and can give you url values if you ever want to put the marker into a link. Anyways it should save you from writing some impls (if you wanna add more features to the class itself later you could always make a class that inherits from it).

Reply:
Done.  This is a good bookmark if we chose to do anything with maps in the capstone.  Right now I'm just going to be using the tuples.